lib/api/src/modules/stories.ts

```diff
@@ -197,6 +198,20 @@ export const init: ModuleFn = ({
         storiesFailed: error,
       });
     },
+    selectFirstStory: () => {
+      const { storiesHash } = store.getState();
+      const lookupList = Object.keys(storiesHash).filter(
+        (k) => !(storiesHash[k].children || Array.isArray(storiesHash[k]))
+      );
+      const firstStory = lookupList[0];
+
+      if (firstStory) {
+        api.selectStory(firstStory);
+        return;
+      }
+
+      navigate('/story/*');
```

Comment:
This doesn't do anything, I don't think. If there's no stories at all I think should just set the URL to `/`.

Reply:
https://github.com/storybookjs/storybook/pull/11360/commits/5e1e0908668e0f4d50ecfe11afc7e29e1b7d86d3